.travis.yml

```diff
@@ -67,11 +67,11 @@ install: |
 
 
 script:
-  - if [ "$TASK" = "core-tests"    ]; then $ANT core-tests    && $ANT serialization-tests && $ANT coverage; fi
+  - if [ "$TASK" = "core-tests"    ]; then $ANT core-tests    && $ANT coverage; fi #  && $ANT serialization-tests
```

Comment:
Should the `serialization-tests` move to the snapshot tests?
Also would ideally have a comment.

And, you probably want to disable the whole `snapshot-tests` job earlier in the file.